src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+:root {
+  --custom-color: rgba(61, 78, 97, 1);
+
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.big-search {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.small-search {
+  margin: 0;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  outline: none;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+
+  font-family: Avenir, sans-serif;
```

Comment:
As best practice define the main `font-family` at a `body` level, so that all the content has it by default